packages/gasket-intl/test/components.test.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import assume from 'assume';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+import { mount } from 'enzyme';
+import { IntlProvider } from 'react-intl';
+import { ERROR, LOADED } from '../src/utils';
+
+const fetchStub = sinon.stub();
```

Comment:
you should put this and the proxyquire call into the `beforeEach` for better test independence